### PR TITLE
Replace "Element" with a brand const

### PR DESCRIPTION
**Pull request by luixxiul**
Addresses https://github.com/vector-im/element-web/issues/21452

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

I am not quite sure if these changes are meaningful. Please let me know if they are not necessary.

Notes: none
element-web notes: none

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://pr8074--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/structures/MatrixChat.tsx [hunk 1]
```diff
@@ -442,16 +442,19 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
...
         const scamText = _t(
             "If someone told you to copy/paste something here, " +
             "there is a high likelihood you're being scammed!",
         );
         const devText = _t(
-            "If you know what you're doing, Element is open-source, " +
-            "be sure to check out our GitHub (https://github.com/vector-im/element-web/) " +
-            "and contribute!",
+            "If you know what you're doing, %(brand)s is open-source, " +
+            "be sure to check out our GitHub (%(repository)s) and contribute!", { brand, repository },
```
[the remark this answers is not in the document]
Comment (luixxiul) [reply src/components/structures/MatrixChat.tsx:457]: I was thinking about the case like this: https://github.com/SchildiChat/matrix-react-sdk/blob/c298521b65be6aa956f93ca6e3a4cfe0189b30e1/src/i18n/strings/en_EN.json#L3104

`"If you know what you're doing, Element is open-source, be sure to check out our GitHub (https://github.com/vector-im/element-web/) and contribute!": "If you know what you're doing, SchildiChat is open-source, be sure to check out our GitHub (https://github.com/vector-im/element-web/) and contribute!"`

Obviously, https://github.com/vector-im/element-web/ is not the GitHub repository of SchildiChat :sweat_smile: Does making the repository link (including `GitHub`)  configurable make sense?

File: src/toasts/MobileGuideToast.ts [hunk 2]
```diff
@@ -43,8 +45,8 @@ export const showToast = () => {
         title: _t("Use app for a better experience"),
         props: {
             description: _t(
-                "Element Web is experimental on mobile. " +
-                "For a better experience and the latest features, use our free native app.",
+                "%(brand)s Web is experimental on mobile. " +
```
[the remark this answers is not in the document]
Comment (luixxiul) [reply src/toasts/MobileGuideToast.ts:48]: Let me check if there are other cases where `Web` is used...

File: src/components/structures/MatrixChat.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (luixxiul) [reply src/components/structures/MatrixChat.tsx:457]: It is not responsibility of vector-im to manage how other people set their repository link, though...

[the remark this answers is not in the document]
Comment (t3chguy) [reply src/components/structures/MatrixChat.tsx:454]: I think for now not tackling this larger issue is most sane

[the remark this answers is not in the document]
Comment (luixxiul) [reply src/components/structures/MatrixChat.tsx:454]: I would agree. Reverting this change for now.

**Pull request merged**